docs/fastdds/statistics/dds_layer/domainParticipant.rst

```diff
@@ -15,9 +15,6 @@ For this purpose, *Fast DDS Statistics module* exposes an extended DDS |Statisti
 Enable statistics DataWriters
 -----------------------------
 
-.. warning::
-    This method will be implemented in future releases. For the moment it always returns ``RETCODE_UNSUPPORTED``.
-
```

Comment:
I'm missing a snippet in this secction

Reply:
I have added a complete code snippet using all the DDS API after explaining the API